terraform/examples/io500/config/config_1c_1s_8d.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
```

Comment:
I can see having 1 config per DAOS Server config. 

If a user wants more clients or server they can do that at the Terraform layer. 

This is is way too many scripts here.